thumbor/engines/pil.py

```diff
@@ -128,6 +135,18 @@ def create_image(self, buffer):
 
         return img
 
+    def get_exif_copyright(self):
+        exifs = piexif.load(self.image.info.get("exif"))
+        copyright_exif = {}
+        if exifs is None or "0th" not in exifs or exifs["0th"].items() is None:
+            return None
+
+        for exif_cod, exif in exifs["0th"].items():
+            if exif_cod in KEEP_EXIF_COPYRIGHT_TAGS:
+                copyright_exif[exif_cod] = exif
```

Comment:
How about iterating over the potentially shorter collection?
```suggestion
        for copyright_tag in KEEP_EXIF_COPYRIGHT_TAGS:
            if copyright_tag in exifs["0th"]:
                copyright_exif[copyright_tag] = exifs["0th"][copyright_tag]
```
This also saves a bit of time — albeit marginal — doing lookups on a dict (constant time) instead of a list (linear).